lib/model.js

```diff
@@ -213,7 +214,7 @@ ModelBaseClass.prototype._initProperties = function(data, options) {
         properties[p] = {name: typeName, type: propType};
         this.setStrict(false);
       }
-
+      */
       // Relation
```

Comment:
I think you are right, this is an other place that just sets the strict false in hardcoded. But it also assigns the relation as a dynamically set property in the model. I could leave this and just comment the `setStrict` statement. I will revisit it